lambda_functions/v1/openapi/deputy-reporting-openapi.yml

```diff
@@ -707,6 +726,11 @@ components:
           type: string
           pattern: '.*'
           description: The base64 encoded file
+        s3_reference:
```

Comment:
Don't add s3_reference in V1 (that's part of IN-235, which we're applying to V2)